### DISCUSSION TITLE
Confusing invoice text for SEPA direct debit orders

### DISCUSSION THREAD

**Discussion by ginkel** [pretix/pretix#4234] [Q&A] [answered]
### Problem and impact

When paying an order using SEPA direct debit, the text "Please complete your payment before {expire_date}." / "Wir bitten um Bezahlung des offenen Betrags bis zum {expire_date}." is still printed on the invoice, which is somewhat confusing as the order is actually marked as paid and no action from the customer is necessary.

### Expected behaviour

It would be nice if the text could be omitted for paid / SEPA direct debit orders.

### Screenshots

![image](https://github.com/pretix/pretix/assets/174129/316eb990-e495-4e95-9eeb-c3526ca9ee89)

### Operating system, dependency versions

Official Docker image on Ubuntu 22.04

### Version

2024.5.0

**Comment by raphaelm** [comment]
What's your setting at Einstellungen → Rechnungswesen → Rechnung ausstellen?

**Comment by ginkel** [comment]
"Automatisch vor Zahlung für alle neuen Bestellungen" the main reason being that we have very few B2B customers who need an invoice to pay...

**Reply by raphaelm** [reply, marked as answer]
Please try " Automatisch nach Zahlung oder wenn die Zahlungsmethode es erfordert" instead.

You can set the checkbox "Generiere Rechnungen für Bestellungen die Bankeinzug als Zahlungsmethode nutzen sofort, auch wenn die Veranstaltung konfiguriert ist Rechnungen erst nach Zahlungseingang zu generieren." in Settings > Payment > Bank transfer to solve the B2B issue (should be active by default on newer events).

**Reply by ginkel** [reply]
Thanks, works like a charm. But shouldn't it read "Generiere Rechnungen für Bestellungen die **Banküberweisung** als Zahlungsmethode nutzen sofort, auch wenn die Veranstaltung konfiguriert ist Rechnungen erst nach Zahlungseingang zu generieren."?

**Reply by raphaelm** [reply]
...yes, you're right!

**Reply by ginkel** [reply]
Allright, ~~I'll submit a PR~~ I fixed the translation via https://translate.pretix.eu/.
